xml/abstract_commands.xml

```diff
@@ -227,12 +263,16 @@
         </field>
         <field name="0" bits="18:17" />
         <field name="write" bits="16">
-            0: Copy data from the memory location specified in {\tt arg1} into
+            <value v="0" name="arg">
```

Comment:
Should these also be "read" and "write"?

Reply:
"arg0" seems better.

Reply:
Done.